generated/nidcpower/nidcpower/session.py

```diff
@@ -4795,17 +4796,36 @@ def fetch_multiple(self, count, timeout=hightime.timedelta(seconds=1.0)):
         Returns:
             measurements (list of Measurement): List of named tuples with fields:
 
+                - **channel** (str)
                 - **voltage** (float)
                 - **current** (float)
                 - **in_compliance** (bool)
 
         '''
         import collections
-        Measurement = collections.namedtuple('Measurement', ['voltage', 'current', 'in_compliance'])
+        Measurement = collections.namedtuple('Measurement', ['channel', 'voltage', 'current', 'in_compliance'])
 
-        voltage_measurements, current_measurements, in_compliance = self._fetch_multiple(timeout, count)
+        voltage_measurements, current_measurements, in_compliances = self._fetch_multiple(timeout, count)
 
-        return [Measurement(voltage=voltage_measurements[i], current=current_measurements[i], in_compliance=in_compliance[i]) for i in range(count)]
+        if self._repeated_capability == '':
+            channel_names = self._get_channel_names(range(self.channel_count))
+        else:
+            first_channel_name = self._get_channel_names([0])[0]
+            channel_names = _converters.convert_channels_repeated_capabilities(
```

Comment:
I _think_ `_get_channel_names` returns channel numbers when "independent channels" is False and FQCNs when it's True.

If the above is true, we need to make sure the format we return is the same for both `if self._repeated_capability == ''` and for the `else` case.

Reply:
I believe these have been covered by my newly added [system tests](https://github.com/ni/nimi-python/pull/1754/files/9d606e85894e6c80b5ed7e87431a2feb2d1d21a0#diff-588aa0a2d97cf60012b577567bec4ae6bad13d2d7201507a924b8aa150bbe328):

```
@pytest.mark.parametrize(
    'resource_name,channels,independent_channels,measurement_channels,expected_measured_channels',
    [
        ('Dev1', None, False, None, [str(x) for x in range(12)]), # "independent channels" is False, self._repeated_capability == '', expected non-FQCNs
        ('Dev1', '0,2-3,6', False, '2-3,0', ['2', '3', '0']), # "independent channels" is False, self._repeated_capability != '', expected non-FQCNs
        ('Dev1', None, True, None, [f'Dev1/{x}' for x in range(12)]), # "independent channels" is True, self._repeated_capability == '', expected FQCNs
        ('Dev1/1:4', ' ', True, '2:3', ['Dev1/2', 'Dev1/3']), # "independent channels" is True, self._repeated_capability != '', expected FQCNs
        ...
    ]
)
```